source/atlas-cli-ephemeral-cluster.txt

```diff
@@ -0,0 +1,115 @@
+.. _atlas-cli-ephemeral-cluster:
+
+=====================================================
+Test Automations with Ephemeral Projects and Clusters
+=====================================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+This tutorial demonstrates how to quickly create and delete ephemeral 
+|service| projects and {+clusters+} using the {+atlas-cli+}. Ephemeral
```

Comment:
I was just going to ask if we could use a simpler word than ephemeral, but explaining it here is a nice touch